packages/opentelemetry-web/src/WebTracerProvider.ts

```diff
@@ -15,7 +15,12 @@
  */
 
 import { BasePlugin } from '@opentelemetry/core';
-import { BasicTracerProvider, TracerConfig } from '@opentelemetry/tracing';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
```

Comment:
Switch to stack

Reply:
Is there any specific reason to use the stack scope manager by default ?

Reply:
Zone scope manager depends on zone.js which may not be available. If we have it as a dependency, then we may have a different zone.js than the end user. If we have it as a peer dependency, the scope manager would be broken for any user that doesn't have zone.js.